config/manager/operator.yaml

```diff
@@ -55,7 +55,7 @@ spec:
             - name: CONFIG_OBSERVABILITY_NAME
               value: config-observability
             - name: KUBERNETES_MIN_VERSION
-              value: ""
+              value: "1.24.0"
```

Comment:
why was it empty before?

Reply:
It was and will be empty. We would like to offer a way to override the kube version. If it is empty, it will default to the value knative/pkg sets.